### PR TITLE
added protobuf serialization example

### PR DESCRIPTION
**Pull request by gAmUssA**
PersonProto serializer and some predicates example

### CODE REVIEW

File: serialization/protobuf-serializer/src/main/java/Member.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        // configure hazelcast serializers to use provided serializer for protobuf
+        final SerializerConfig personProtoSerializerConfig = new SerializerConfig();
+        personProtoSerializerConfig.setTypeClass(Person.class);
+        personProtoSerializerConfig.setImplementation(new PersonProtoSerializer());
+        config.getSerializationConfig().addSerializerConfig(personProtoSerializerConfig);
+        // end serialization config
+
+        final HazelcastInstance hazelcastInstance = newHazelcastInstance(config);
+        final IMap<Integer, Person> personsMap = hazelcastInstance.getMap("person");
+        populatePerson(personsMap);
```
Comment (pveentjer) [inline serialization/protobuf-serializer/src/main/java/Member.java]: I would call this just 'populate'. Persons is already implied by the map name and populatePerson sounds like a single person being added.

File: serialization/protobuf-serializer/src/main/java/PersonProtoSerializer.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    @Override
+    public PersonProtos.Person read(ObjectDataInput in) throws IOException {
+        return PersonProtos.Person.parseFrom(in.readByteArray());
+    }
+
+    @Override
+    public int getTypeId() {
+        //
```
Comment (pveentjer) [inline serialization/protobuf-serializer/src/main/java/PersonProtoSerializer.java]: why the // ?

**Comment by pveentjer** [comment]
Minor comments. Nothing obstructing a merge.

**Comment by gAmUssA** [comment]
@pveentjer resolved

**Comment by gAmUssA** [comment]
@mesutcelik it's done

**Pull request merged**